src/formatter.rs

```diff
@@ -187,22 +187,33 @@ pub fn print_logs(log_query: &StorageLogQuery) {
     log::info!("{}", separator);
 }
 
-pub fn print_vm_details(result: &VmPartialExecutionResult) {
+pub fn print_vm_details(result: &VmExecutionResultAndLogs) {
     log::info!("");
-    log::info!("┌──────────────────────────┐");
+    log::info!("\n┌──────────────────────────┐");
```

Comment:
No need for new lines here anymore as we have `log::info!("")` above
```suggestion
    log::info!("┌──────────────────────────┐");
```